codacy/values-production.yaml

```diff
@@ -1,14 +1,16 @@
 global:
   github: {}
-    # if you set this provider configuration, you must save it on the admin/integrations ui.
+    # If you set this provider configuration, you must save it on the admin/integrations UI.
```

Comment:
does this still apply? there is no longer any configuration in the UI for github

Reply:
You're right, I'm removing this line. :+1: